websqrl/src/bluesky-sample.sqrl

```diff
@@ -14,16 +14,16 @@ LET DatasetId := 1;
 LET ProfanityFilterEnabled := false;
 
 # Feature extraction from the Twitter data
-LET TweetText := jsonValue(EventData, '$.data.text');
-LET AuthorId := jsonValue(EventData, '$.data.author_id');
-LET TweetId := jsonValue(EventData, '$.data.id');
-LET TweetDate := jsonValue(EventData, '$.data.created_at');
-LET Users := jsonValue(EventData, '$.includes.users');
-LET AuthorUserData := first([It for It in Users where jsonValue(It, "$.id") = AuthorId]);
-LET AuthorUsername := entity("User", jsonValue(AuthorUserData, "$.username"));
-LET AuthorCreatedAt := jsonValue(AuthorUserData, "$.created_at");
-LET AuthorName := jsonValue(AuthorUserData, "$.name");
-LET AuthorProfileImageUrl := jsonValue(AuthorUserData, "$.profile_image_url");
+LET Version := jsonValue(EventData, '$.v');
+LET EventName := jsonValue(EventData, '$.eventName');
+LET RecordType := jsonValue(EventData, '$.payload.record.$type');
+LET AuthorId := jsonValue(EventData, '$.payload.author');
+LET TweetText := jsonValue(EventData, '$.payload.record.text');
+LET TweetId := jsonValue(EventData, '$.payload.cid');
+# TODO(meyer) payload.record.createdAt is not available on all events
```

Comment:
Is this an issue? Fine to leave this in and figure it out later though.

Reply:
yeh, i noticed it on `delete-like` events but there might be others as well